### ISSUE TITLE
Add required and aria-required attributes to forms & improve overall accessibility

### ISSUE THREAD

**Issue by a-fink** [a-fink/authenticate-me#4]
Requirements:
- all input fields in forms should have 'required' and 'aria-required' attributes
- test and confirm that all forms are navigable using mouse and keyboard (tab / enter etc)
- ensure all images have 'alt' and 'title' attributes
